### PR TITLE
JAMES-3394 Set guice docker jdk.tls.client.protocols to "TLSv1,TLSv1.…

### PR DESCRIPTION
**Pull request by chibenwa**
…1,TLSv1.2"

This is intended to solve `javax.net.ssl.SSLException: Session has no PSK` and
relax the default TLS set up of Java 8 (TLSv1.2)

### CODE REVIEW

File: dockerfiles/run/guice/jpa-smtp/destination/run_james.sh [hunk 1]
```diff
@@ -2,4 +2,7 @@
 if [ "$GLOWROOT_ACTIVATED" == "true" ]; then
     GLOWROOT_OPTIONS=-javaagent:/root/glowroot/glowroot.jar
 fi
-java -classpath '/root/james-server-jpa-smtp-guice.jar:/root/james-server-jpa-smtp-guice.lib/*' -javaagent:/root/james-server-jpa-smtp-guice.lib/openjpa-3.1.0.jar $GLOWROOT_OPTIONS -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ $JVM_OPTIONS org.apache.james.JPAJamesServerMain
\ No newline at end of file
+java -Dlogback.configurationFile=/root/conf/logback.xml \
```
Comment (Arsnael) [inline dockerfiles/run/guice/jpa-smtp/destination/run_james.sh:5]: the command before was a bit different with more arguments... Is it just something outdated or something got forgotten here?

File: dockerfiles/run/guice/jpa/destination/run_james.sh [hunk 2]
```diff
@@ -2,4 +2,7 @@
 if [ "$GLOWROOT_ACTIVATED" == "true" ]; then
     GLOWROOT_OPTIONS=-javaagent:/root/glowroot/glowroot.jar
 fi
-java -classpath '/root/james-server.jar:/root/james-server-jpa-guice.lib/*' -javaagent:/root/james-server-jpa-guice.lib/openjpa-3.1.0.jar $GLOWROOT_OPTIONS -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ $JVM_OPTIONS org.apache.james.JPAJamesServerMain
+java -Dlogback.configurationFile=/root/conf/logback.xml \
```
Comment (Arsnael) [inline dockerfiles/run/guice/jpa/destination/run_james.sh:5]: same question

File: dockerfiles/run/guice/jpa-smtp/destination/run_james.sh (hunk 1 above)
Comment (chibenwa) [reply dockerfiles/run/guice/jpa-smtp/destination/run_james.sh:5]: OUPS

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
`Caused by: java.net.BindException: Address already in use`

test this please

**Comment by Arsnael** [comment]
test this please

**Review by rouazana — COMMENTED** [review]
should we really allow tlsv1 and tlsv1.1?

**Comment by chibenwa** [comment]
> should we really allow tlsv1 and tlsv1.1?

Maybe not. TLS 1.1 [got removed from chrome](https://medium.com/@simturax/google-chrome-84-is-available-removes-support-for-tls-1-0-and-tls-1-1-90000f2a90fa), [firefox](https://www.fxsitecompat.dev/en-CA/docs/2020/tls-1-0-1-1-support-has-been-disabled-by-default/) and  [edge](https://blogs.windows.com/msedgedev/2020/03/31/tls-1-0-tls-1-1-schedule-update-edge-ie11/).

Parties interested in that support could override the run command.

At Linagora, we should be able to live with these logs.

**Pull request closed without merging**